.circleci/config.yml

```diff
@@ -942,9 +979,8 @@ jobs:
                       name: Restore cached uberjar built in previous step
                       <<: *CacheKeyUberjar
                   - steps: << parameters.before-steps >>
-                # Make both `test-files` and `source-folder` parameters optional. Translates to: if `parameter` => run associated flag (`--spec` and `--folder`, respectively)
                 command: |
-                  run test-cypress-no-build <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> <<# parameters.source-folder >> --folder << parameters.source-folder >> <</ parameters.source-folder >>
+                  run test-cypress-no-build --folder << parameters.source-folder >>  --record --key $CURRENTS_KEY --group << parameters.source-folder >> --ci-build-id $CIRCLE_SHA1
```

Comment:
Why do we need a custom `--ci-build-id`?

Reply:
Commented here https://github.com/metabase/metabase/pull/20997#issuecomment-1067907949. But need ajustments

Reply:
We are not using parallelization. That's why I was confused.

https://currents.dev/readme/concepts/cypress-ci-build-id#what-is-cypress-ci-build-id
> CI Build ID is a unique build identifier used by Currents to distinguish parallel cypress test runs one from another. Multiple CI machines using the same CI Build ID will belong to the same test run.

Reply:
I'll remove `--ci-build-id` and just let `--group`

About spec there's a thread here: https://github.com/metabase/metabase/pull/20997#discussion_r826897681